src/lib/helpers.js

```diff
@@ -0,0 +1,23 @@
+import { t } from '../lib/i18n'
+
+// Take an items array and return an array of category objects with the matching title and items
+export function getCategorizedItems (items) {
+  if (items[0] instanceof Array) return null // doesn't handle this case
+  const categorizedItemsObject = items.reduce((accumulator, item) => {
+    accumulator[item.category] = accumulator[item.category] || []
+    accumulator[item.category].push(item)
+    return accumulator
+  }, {})
+
+  return Object.keys(categorizedItemsObject)
+    .map(category => {
+      return {title: category, items: categorizedItemsObject[category]}
```

Comment:
Could we use `slug` instead of `title` ? In my opinion `title` is more a "human readable" property, which may be displayed to user, and slug a technical one. What do you think ?